index.js

```diff
@@ -0,0 +1,17 @@
+const navEl = document.querySelector('.nav-items')
+const closeEl = document.querySelector('.close-btn');
+const hamburgerEl = document.querySelector('.menu-icon');
+const linksEl = document.querySelectorAll('.nav-links');
```

Comment:
- [ ] As [required ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md)for this project, your project is still missing the following requirements: 

 - When the user clicks (or taps) the hamburger button, the mobile menu appears.
 - When the user clicks (or taps) the close (X) button, the mobile menu disappears.
 - When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
 - When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

Please kindly implement the above changes

Reply:
@Mihndim2020 
kindly explain further, from my end all those functions work as required.

Reply:
Hi @Mihndim2020,
kindly clarify further, I don't understand the issue apart from the readme file update.